src/org/scada_lts/dao/MultiChangeHistoryDAOImp.java

```diff
@@ -0,0 +1,94 @@
+package org.scada_lts.dao;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.scada_lts.dao.model.multichangehistory.MultiChangeHistoryValues;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.util.List;
+
+/**
+ * @author grzegorz.bylica@abilit.eu on 16.10.2019
+ */
+@Repository
+public class MultiChangeHistoryDAOImp implements MultiChangesHistory {
+
+    private static final Log LOG = LogFactory.getLog(MultiChangeHistoryDAOImp.class);
+
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_ID = "mch.id";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_ID = "mch.userId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_NAME = "mch.username";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_AND_CMP_IDENTIFICATION = "mch.viewAndComponentIdentification";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_INTERPRETED_STATE = "mch.interpretedState";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_TS = "mch.ts";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE_ID = "vmch.valueId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE = "vmch.value";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_DATA_POINT_ID = "vmch.dataPointId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_XID_POINT = "xidPoint";
```

Comment:
Is it possible to create additional String variable and disconnect chains "mch", vmch" to separated variables? Maybe if is possible same situation we would like to do with dot char -> separated variable like just DOT. Variables which have only columns names, they could have only columns names-just something like "username". As example : COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_XID_POINT is great.

Reply:
improved